File: programmers/lv0/코딩기초트레이닝/조건문/이어붙인수/Solution.cs
using System;

public class Solution {
    public int solution(int[] num_list)
        {
            string even = "";
            string odd = "";
            foreach (var x in num_list)
            {
                if (x % 2 == 0) even += x;
                else odd += x;
            }

            return Convert.ToInt32(even) + Convert.ToInt32(odd);
        }
}

File: programmers/lv0/코딩기초트레이닝/조건문/이어붙인수/Solution.go
package main

import "fmt"

func solution(num_list []int) int {
	even := 0
	odd := 0
	for _, num := range num_list {
		if num%2 == 0 {
			even = even*10 + num // 8
		} else {
			odd = odd*10 + num // 5 -> 50+7=57 -> 570+3=573
		}
	}
	return even + odd
}
func main() {
	fmt.Println(solution([]int{3, 4, 5, 2, 1}))
	fmt.Println(solution([]int{5, 7, 8, 3}))
}
